demo/site/preBuild/src/createRedirects.ts

```diff
@@ -45,26 +45,26 @@ const createApiRedirects = async (): Promise<Redirect[]> => {
     const redirects: Redirect[] = [];
 
     for (const redirect of response.redirects) {
-        let source: string | undefined;
-        let destination: string | undefined;
+        let source: string | null = null;
+        let destination: string | null = null;
 
         if (redirect.sourceType === "path") {
             source = redirect.source;
         }
 
         const target = redirect.target as RedirectsLinkBlockData;
 
-        if (target.block !== undefined) {
+        if (target.block) {
             switch (target.block.type) {
                 case "internal":
-                    destination = (target.block.props as InternalLinkBlockData).targetPage?.path;
+                    destination = (target.block.props as InternalLinkBlockData).targetPage?.path || null;
```

Comment:
```suggestion
                    destination = (target.block.props as InternalLinkBlockData).targetPage?.path ?? null;
```

This would be the "more correct" fallback for optional chaining